src/Microsoft.ML.Analyzer/TypeIsSchemaShapeAnalyzer.cs

```diff
@@ -2,420 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.Generic;
 using System.Collections.Immutable;
-using System.Linq;
 using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Diagnostics;
 
 namespace Microsoft.ML.Analyzer
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class TypeIsSchemaShapeAnalyzer : DiagnosticAnalyzer
```

Comment:
What's the value of this empty Analyzer? Shouldn't it just be deleted?

Reply:
You are right, I though that this would give problems with the API compat since it is a public class, but I removed it altogether as you were suggesting and it does not complain. I guess it's because it's an analyzer!

---
In reply to: [304492042](https://github.com/dotnet/machinelearning/pull/4009#discussion_r304492042) [](ancestors = 304492042)